hooks/useZoraPremint.tsx

```diff
@@ -15,44 +15,95 @@ const useZoraPremint = () => {
   const { sendTransaction: sendTxByWallet } = useWalletSendTransaction()
   const { connectWallet } = usePrivy()
 
+  const IS_TESTNET = process.env.NEXT_PUBLIC_TESTNET === "true"
+
+  const proofs = {
```

Comment:
can you please move this into a standalone `json` file?
maybe something like `/lib/whitelistProof.json`

Reply:
Added this comment changes to the PR.